packages/library-base/src/commonMain/kotlin/io/realm/internal/BaseRealmImpl.kt

```diff
@@ -62,6 +64,14 @@ abstract class BaseRealmImpl internal constructor(
         log.info("Realm opened: ${configuration.path}")
     }
 
+    // FIXME Currently constructs a new instance on each invocation. We could cache this pr. schema
```

Comment:
I would probably prefer caching it. It should also be possible inside the Notifier thread.

Reply:
Caching it requires that it is update along with the `RealmReference` to work on Kotlin native. I think implementing this goes better in hand with implementation of key caching. Created #567 to track that.